src/communication/dto/communication.dto.ts

```diff
@@ -0,0 +1,49 @@
+import { Role } from "@prisma/client";
+import {
+   IsDateString,
+   IsNotEmpty,
+   IsOptional,
+   IsString,
+} from 'class-validator';
+
+export class updateEventDto{
```

Comment:
Your DTOs should use Pascal Case. So change this to `UpdateEventDto`. Same thing for every class in this file.